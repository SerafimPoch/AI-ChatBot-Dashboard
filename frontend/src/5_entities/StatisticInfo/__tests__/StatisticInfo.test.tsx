import React from "react";
import { render } from "@testing-library/react";
import {
  StatisticInfoContainer,
  StatisticInfoTitle,
  StatisticInfoValue,
  StatisticInfoNumbersWrapper,
} from "../StatisticInfo.styles";

describe("StatisticInfo page components", () => {
  it("StatisticInfoContainer renders correctly", () => {
    const { asFragment } = render(<StatisticInfoContainer />);
    expect(asFragment()).toMatchSnapshot();
  });
  it("StatisticInfoTitle renders correctly", () => {
    const { asFragment } = render(<StatisticInfoTitle />);
    expect(asFragment()).toMatchSnapshot();
  });
  it("StatisticInfoValue renders correctly", () => {
    const { asFragment } = render(<StatisticInfoValue />);
    expect(asFragment()).toMatchSnapshot();
  });
  it("StatisticInfoNumbersWrapper renders correctly", () => {
    const { asFragment } = render(<StatisticInfoNumbersWrapper />);
    expect(asFragment()).toMatchSnapshot();
  });
});
