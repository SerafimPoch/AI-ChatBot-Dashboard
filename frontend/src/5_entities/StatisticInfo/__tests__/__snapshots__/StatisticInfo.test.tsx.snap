// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StatisticInfo page components StatisticInfoContainer renders correctly 1`] = `
<DocumentFragment>
  .emotion-0 {
  padding: 24px 36px;
  display: grid;
  grid-template-columns: repeat(3, 1fr);
  gap: 24px;
  border-radius: 24px;
  border: 1px solid rgba(255, 255, 255, 0.7);
  background-color: undefined;
  -webkit-backdrop-filter: blur(24px);
  backdrop-filter: blur(24px);
}

@media (max-width: 1000px) {
  .emotion-0 {
    display: none;
  }
}

<div
    class="emotion-0"
  />
</DocumentFragment>
`;

exports[`StatisticInfo page components StatisticInfoNumbersWrapper renders correctly 1`] = `
<DocumentFragment>
  .emotion-0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  gap: 5px;
}

<div
    class="emotion-0"
  />
</DocumentFragment>
`;

exports[`StatisticInfo page components StatisticInfoTitle renders correctly 1`] = `
<DocumentFragment>
  .emotion-0 {
  font-family: var(--font-sf-pro-rounded);
  font-size: 18px;
  font-weight: 600;
  line-height: 24px;
  color: undefined;
  margin-bottom: 8px;
}

<p
    class="emotion-0"
  />
</DocumentFragment>
`;

exports[`StatisticInfo page components StatisticInfoValue renders correctly 1`] = `
<DocumentFragment>
  .emotion-0 {
  font-family: var(--font-sf-pro-display);
  color: undefined;
  font-size: 56px;
  font-weight: 700;
  line-height: 64px;
  letter-spacing: -0.84px;
}

<p
    class="emotion-0"
  />
</DocumentFragment>
`;
